recipe/meta.yaml

```
@@ -1,30 +1,28 @@
-{% set version = "7.4.2" %}
+{% set version = "8.9.0" %}
 
 package:
   name: gradle
   version: {{ version }}
 
 source:
   url: https://github.com/gradle/gradle/archive/v{{ version }}.tar.gz
-  sha256: 4cbfde441a8afd96a8503a120402839838e3d6bad95aed5d015e642f9c40668c
+  sha256: 9172bbdc47265e35ced086c48a56b39d52376e1e1b4417d38583b4e4b25f6d2f
 
 build:
   number: 0
+  noarch: generic
 
 requirements:
-  build:
-    - {{ compiler('c') }}
-    - sysroot_linux-64 2.17  # [linux64]
   host:
-    - openjdk >=11
+    - openjdk >=11,<12
```

Can you document where this restriction comes from?
```suggestion
    # ...
    - openjdk >=11,<12
```

---

Done